modules/navier_stokes/doc/content/source/auxkernels/Courant.md

```diff
@@ -0,0 +1,25 @@
+# Courant
+
+!syntax description /AuxKernels/Courant
+
+## Overview
+
+The `Courant` object computes the Courant number given coupled variables
+representing velocity components (`u`, `v`, and `w`) and using the current
+timestep size and element size. The Courant number formula is:
+
+\begin{equation}
+C = \frac{u\Delta t}{h}
```

Comment:
I believe it's the sum u/h + v/hy + w/hz
but really it depends on the numerical scheme used.

did this one work well for cnsfv?

Reply:
In general for explicit time integration I found that the time step size had to be significantly less than the Courant number, or even 0.5 times the Courant number in order to avoid oscillations. Significantly meaning at least an order of magnitude less